ext/intl/bidi/bidi.c

```diff
@@ -228,15 +273,17 @@ static PHP_METHOD(IntlBidi, setContext) {
 	UChar *uprologue = NULL, *uepilogue = NULL;
 	int32_t uprologue_len = 0, uepilogue_len = 0;
 
-	if (zend_parse_parameters_throw(ZEND_NUM_ARGS(), "|S!S!", &prologue, &epilogue) == FAILURE) {
-		return;
-	}
+	ZEND_PARSE_PARAMETERS_START(0, 2)
+		Z_PARAM_OPTIONAL
+		Z_PARAM_STR_EX(prologue, 0, 0)
```

Comment:
check_null arg should be 1 to match the '!', not 0.

```suggestion
		Z_PARAM_STR_EX(prologue, 1, 0)
```

Should also update the ZEND_ARG_TYPE_INFO() lines above to match that "nullable" concept:

```
    ZEND_ARG_TYPE_INFO(0, prologue, IS_STRING, 1)
```

Reply:
Fixed it.